docs/build.rb

```diff
@@ -357,6 +357,9 @@ def fix_github_username_links(content)
         '@git_addr',
         '@git_objs',
         '@refs',
+        '@conf_content',
+        '@need_cleanup',
+        '@repo_name'
```

Comment:
Instead of updating this build list that affects all of the builds, would it not be better to just change the Ruby example code to not use instance variables? You could change it to `self.conf_content`, `conf_content` or even a literal value. `@need_cleanup` as an instance variable seems to be a specific convention to two moduless.

Reply:
Sounds good, I'll make these changes. I didn't know these build settings were so impactful. In my understanding, they were just here to avoid false positive when the build process tries to automatically add the Github URL to any word starting with `@`. I'm wondering if there is a way to improve the regex and avoid matching instance variable in example code surrounded by \`\`\` markdown.

Reply:
I've added a PR to remove this functionality https://github.com/rapid7/metasploit-framework/pull/17849 - as we're not using too many Github handles in docs in the first place, it was just an artifact of the Github wiki migration that I initially implemented to make the transition easier

> I'm wondering if there is a way to improve the regex and avoid matching instance variable in example code surrounded by ``` markdown.

We should be using a real markdown parser not Regex, but the Ruby markdown parsers as flexible in comparison to the ones over in the node community IMO